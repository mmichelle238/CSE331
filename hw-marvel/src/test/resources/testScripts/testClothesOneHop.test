# Find a path with only one "hop" (edge) using a file I created
# called newClothes.

LoadGraph clothes newClothes.tsv

FindPath clothes pants shorts