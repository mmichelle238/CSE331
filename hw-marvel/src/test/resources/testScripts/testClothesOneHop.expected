# Find a path with only one "hop" (edge) using a file I created
# called newClothes.

loaded graph clothes

path from pants to shorts:
pants to shorts via pantbook